notification-bot/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "apibara-server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon --exec ts-node src/index.ts"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@apibara/protocol": "^0.4.9",
+    "@apibara/starknet": "^0.5.0",
+    "axios": "^1.6.8",
```

Comment:
axios isn't used anywhere in the project I think?

Reply:
done